nbextensions/appCell2/widgets/appCellWidget.js

```diff
@@ -1786,9 +1786,9 @@ define([
                     // to inform us about what processing stage the
                     // error occurred in -- we need to inspect the job state.
                     var errorStage;
-                    if (jobState.exec_start_time) {
+                    if (jobState.running) {
```

Comment:
Can this be figured out by the job status? And not guessing from which fields are present?

Reply:
hmm it's already in `jobState.status` case block.